ckanext/canada/tables/aistrategy.yaml

```diff
@@ -0,0 +1,818 @@
+    import_template_include: false
+    visible_to_public: false
+    datastore_type: text
+    preview_class: bg-info
+
+  datastore_primary_key: [ref_number, reporting_period]
+  datastore_indexes: ""
+
+  default_preview_sort: ref_number, reporting_period desc, priority_area
+
+  cascading_select_fields: true
+
+  excel_example_height: 50
+
+  triggers:
+  - aistrategy_trigger: |
+      DECLARE
+        errors text[][] := '{{}}';
+        crval RECORD;
+      BEGIN
+        errors := errors || required_error(NEW.ref_number, 'ref_number');
+
+        errors := errors || required_error(NEW.reporting_period, 'reporting_period');
+        errors := errors || choice_error(NEW.reporting_period, {reporting_period}, 'reporting_period');
+
+        errors := errors || required_error(NEW.priority_area, 'priority_area');
+        errors := errors || choice_error(NEW.priority_area, {priority_area}, 'priority_area');
+
+        errors := errors || required_error(NEW.key_action, 'key_action');
+        errors := errors || choice_error(NEW.key_action, {key_action}, 'key_action');
+
+        errors := errors || required_error(NEW.sub_action, 'sub_action');
+        errors := errors || choice_error(NEW.sub_action, {sub_action}, 'sub_action');
+
+        errors := errors || required_error(NEW.activity, 'activity');
+        errors := errors || choice_error(NEW.activity, {activity}, 'activity');
+
+        NEW.description_en := trim_lead_trailing(NEW.description_en);
+        errors := errors || max_char_error(NEW.description_en, 1000, 'description_en');
+
+        NEW.description_fr := trim_lead_trailing(NEW.description_fr);
+        errors := errors || max_char_error(NEW.description_fr, 1000, 'description_fr');
+
+        errors := errors || required_error(NEW.expected_completion, 'expected_completion');
+        errors := errors || choice_error(NEW.expected_completion, {expected_completion}, 'expected_completion');
+
+        errors := errors || required_error(NEW.lead_department, 'lead_department');
+        errors := errors || choice_error(NEW.lead_department, {lead_department}, 'lead_department');
+
+        errors := errors || required_error(NEW.status, 'status');
+        errors := errors || choice_error(NEW.status, {status}, 'status');
+
+        NEW.progress_en := trim_lead_trailing(NEW.progress_en);
+        errors := errors || max_char_error(NEW.progress_en, 1000, 'progress_en');
+
+        NEW.progress_fr := trim_lead_trailing(NEW.progress_fr);
+        errors := errors || max_char_error(NEW.progress_fr, 1000, 'progress_fr');
+
+        IF errors = '{{}}' THEN
+          RETURN NEW;
+        END IF;
+        RAISE EXCEPTION E'TAB-DELIMITED\t%', array_to_string(errors, E'\t');
+      END;
+  - update_record_modified_created_trigger
+
+  examples:
+    record:
+      ref_number: "TBS-001"
+      reporting_period: "2025-Q3"
+      priority_area: "A"
+      key_action: "A.1"
+      sub_action: "A.1.1"
+      activity: "1.3.1"
```

Comment:
```suggestion
      activity: "A.1.3.1"
```

Reply:
add a basic test module that covers loading the example record at least, even better would be testing specific error cases. See test_nap5.py for minimal tests and test_service.py for some extensive ones.

Reply:
This should fix the example data test failure, the fix below is just for the docs